catalog/batch_rename.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
This is a file committed from another PR. Not sure how it appeared here.

Reply:
I merged with Andrew's fork without thinking about it. We can merge the other PR first.